src/components/views/elements/SettingsFlag.tsx

```diff
@@ -106,6 +107,9 @@ export default class SettingsFlag extends React.Component<IProps, IState> {
                     <label className="mx_SettingsFlag_label">
                         <span className="mx_SettingsFlag_labelText">{ label }</span>
                         { description && <div className="mx_SettingsFlag_microcopy">
+                            { shouldWarn && <span className="mx_SettingsTab_microcopy_warning">
+                                { _t("WARNING:") } { " " }
```

Comment:
The warning text should be part of the translation

Reply:
Sorry, I am not sure what you mean here?

Reply:
`_t("<w>WARNING:</w> <description />")` or similar - the `<w>` gets replaced by the span tag. This is for RTL and similar languages where the sentence structure itself can change depending on where the label is.

Reply:
Oh, I see now!